src/patternfly/patternfly-date-picker.scss

```diff
@@ -0,0 +1,177 @@
+// stylelint-disable
+.pf-c-date-picker__calendar {
+  &.flatpickr-calendar {
+    // calendar
+    --pf-c-date-picker__flatpickr__calendar--Background: var(--pf-global--BackgroundColor--light-100);
+    --pf-c-date-picker__flatpickr__calendar--Padding: var(--pf-global--spacer--md);
+    --pf-c-date-picker__flatpickr__calendar--open--ZIndex: var(--pf-global--ZIndex--sm);
+
+    // month/year section
+    --pf-c-date-picker__flatpickr__months--PaddingBottom: var(--pf-global--spacer--md);
+    --pf-c-date-picker__flatpickr__month--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
```

Comment:
why does this need a background color too ?

Reply:
Flatpickr comes with a grey bg there 
<img width="405" alt="Screen Shot 2020-09-22 at 5 52 11 PM" src="https://user-images.githubusercontent.com/35148959/93945358-5bc7cb80-fcfc-11ea-9c55-c482081f76da.png">